fecgov/fec-cms
# Update sitemap-pdf for changes to files

## Summary (required)

Updates the site map to pick up on changes made to the PDFs in it.

## How to test

- verify YYYY-MM-DD format is the one to use for pub dates. Dates should be in April or March 2020.
